src/webapp/components/multiple-selector/MultipleSelector.tsx

```diff
@@ -153,21 +157,42 @@ const useStyles = makeStyles((theme: Theme) =>
                 maxWidth: "100%",
             },
         },
+        label: {
+            maxHeight: "1em",
+            textOverflow: "ellipsis",
+            textWrap: "nowrap",
+            maxWidth: "100%",
+            whiteSpace: "nowrap",
+            overflow: "hidden",
+        },
+        hide: {
+            display: "none",
+        },
     })
 );
 
-const ITEM_HEIGHT = 36; // line-height: 1.5 + padding: 6 * 2
+function getMenuProps(autoFocus: boolean, dividers = 0, isSearchable = false) {
```

Comment:
We tend not to use this args style, instead, create an options object (with required and optional keys)